packages/notices/src/store/actions.ts

```diff
@@ -2,48 +2,22 @@
  * Internal dependencies
  */
 import { DEFAULT_CONTEXT, DEFAULT_STATUS } from './constants';
+import type { NoticeOptions, ReducerAction } from './types';
 
 /**
- * @typedef {Object} WPNoticeAction Object describing a user action option associated with a notice.
+ * @typedef {import('./types').NoticeAction} NoticeAction Notice action object.
  *
- * @property {string}    label   Message to use as action label.
- * @property {?string}   url     Optional URL of resource if action incurs
- *                               browser navigation.
- * @property {?Function} onClick Optional function to invoke when action is
- *                               triggered by user.
+ * @typedef {import('./types').NoticeOptions} NoticeOptions Notice action object.
```

Comment:
I assume these JSDoc type definitions can now get removed. I see that `NoticeOptions` type is imported. `NoticeAction` is no longer referenced in the file.

Reply:
Oh yeah. Done. Thanks